### DISCUSSION TITLE
Submission Directions – Read Before Posting 👀️

### DISCUSSION THREAD

**Discussion by cassidyjames** [elementary/edw#2] [Announcements]
elementary Developer Weekend (edw) is an online conference for developers, designers, and more building apps for elementary OS.

- :octocat: **Submitting requires being signed into a free GitHub account**. Go ahead and do that now if you haven't!

- 🎓️ **Talks should be primarily educational**: your audience should come away from your talk with new skills or perspectives. This is your opportunity to share your expertise with others

- ⌨️ **Talks can be very technical** (it's a developer conference after all), but **introductory level talks are important to have** as well; edw should be seen as an outreach event where we expand and strengthen our developer community.

- 📣️ **Talks which are development adjacent are also welcome**. Design, internationalization, accessibility, and even entrepreneurship are all skills to be honed as well-rounded app developers.

- 🕒️ **Talks should be around 30 minutes**, not including time for Q&A.

- 📏️ Talks must produce results that **adhere to the [AppCenter publishing requirements](https://docs.elementary.io/develop/appcenter/publishing-requirements)**, with the addition of Flatpak as an expectation for elementary OS 6 and beyond.

---

This is a place both to submit your own proposals for talks you would like to present, as well as a place to propose talks you wish existed. 

**Talks** are proposals for presentations that you would like to give; these should be well thought out and you should be committed to preparing and recording a talk, as well as being present during the conference for any Q&A. To submit a talk, [start a new discussion](https://github.com/elementary/edw/discussions/new) in the ["Talk Proposals"](https://github.com/elementary/edw/discussions/categories/talk-proposals) category.

**Ideas** are concepts for talks you'd like to hear or think would be beneficial to the community, but that you are not necessarily committed to giving yourself. For these, use the ["Ideas"](https://github.com/elementary/edw/discussions/categories/ideas) category. We can always promote an Idea to a Talk, so if you're in doubt, submit it as an Idea.

A good proposal should include:

- [x] An engaging title
- [x] The name of the speaker or TBD if the talk needs a speaker
- [x] About a tweet's length explanation of the goal of the talk

**Comment by kmwallio** [comment]
> Talks must produce results that adhere to the AppCenter publishing requirements.

I was going to propose a talk that I wished existed for "Flatpak and elementaryOS 6" or "App Center For Everyone", but the publishing requirements still require Debian Packaging.

Is edw planned for the current state of the world and elementaryOS, or should talks on current/future endeavors also take place?

**Reply by danirabbit** [reply]
I think we can assume a Flatpak world for the purposes of EDW! So those are great suggestions :)

**Reply by cassidyjames** [reply]
Good point! I updated the original post as well.
